components/GameScreen.vue

```diff
@@ -436,6 +483,11 @@ watch(gameState, async (newState, prevState) => {
       class="flex flex-col shadow ml-2"
       :style="{ maxWidth: gameState?.width + 'px' }"
     >
+      <div class="flex flex-row justify-end mb-2 mt-1 mx-4 gap-6">
+        <ButtonLink @click="followBot">
```

Comment:
nit: since it's a single function that toggles the mode on/off, let's reflect this in the function name

```suggestion
        <ButtonLink @click="toggleFollowMeMode">
```